samples/benchmarks/coremark/Kconfig.sysbuild

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+source "share/sysbuild/Kconfig"
+
+config IMAGE_2_BOARD
```

Comment:
Why is all that complexity needed? Zephyr images are meant to be built for a very particular platform and samples & tests are as platforms agnostic as it could be. I.e. user may select whatever combo of board+test he or she prefers.

Reply:
That I don't understand really. IMHO all the platform-specific things need to belong to that platform. I guess the same could be done for any other test, right? In that sense for your users it will be beneficial to get that functionality from your Zephyr platform/board. At the same time users of that sample on another platform don't want that complexity. Not to mention that simpler code is much easier to maintain.